### DISCUSSION TITLE
how can I install Windwso App Sdk

### DISCUSSION THREAD

**Discussion by Javier118** [microsoft/WindowsAppSDK#1033] [Q&A] [answered]
how can I install Windows App Sdk . Windows 10 2004 21H1

**Comment by andrewleader** [comment, marked as answer]
Hi @Javier118, please see the get started docs: https://docs.microsoft.com/windows/apps/windows-app-sdk/get-started

Thanks!

**Reply by Javier118** [reply]
In That Visual Studio Can Install SDK Serves VS Preview 2022
